pip-wheel-metadata/rsconnect_python-1.20.1.dev23+g917b738.d20231017.dist-info/LICENSE.md

```diff
@@ -0,0 +1,361 @@
+### GNU GENERAL PUBLIC LICENSE
```

Comment:
This directory seems like it should be gitignored.

Reply:
added to .gitignore